crypto_guidelines.md

```diff
@@ -47,7 +47,7 @@ in depth rather than due to any particular known exploit techniques.
 ### 1) Do not conditionally choose between constant and non-constant time
 
 It may be natural in crypto code to do the equivalent of:
-```
+```cpp
```

Comment:
My reading is that these should be `c` or `c++`? But I'll admit I'm not a github / markdown expert.

Reply:
I pointed out the dialect of the `c` syntax highlighting works.